### DISCUSSION TITLE
RowSpan

### DISCUSSION THREAD

**Discussion by TheRealNapalm** [richard-scryber/scryber.core#65] [Ideas]
Hi,
I realized that tables support colspan, but do not seem to support rowspan.
Is there a plan to include this one day ?

scryber.core is great.

regards
Mirko

**Comment by richard-scryber** [comment]
Hi Mirko

Thanks, really glad you like Scryber. 

I do hope we can support rowspan in the future - it just hurts my head at the moment, to get the calculations sorted. :-)

At the moment, nested tables may work for you, but the overflow onto new pages could be a blocker.
If there is anything else you can think would make Scryber better, be lovely to hear.

Kind regards

Richard

**Comment by TheRealNapalm** [comment]
Hi Richard,

thank you for your reply.

 

Yes, I solved my little problem with a nested table.

All fine for now for me 😉

 

 

Btw, I found another bug/glitch/whatever

 

I have to create a monospaced font page with data

Also with a lot of whitspaces. like

 

A  b  c  d

==========

1  2     3
   4  5  6

1    2 

1  2  3  4 

just like tables, but as I get the data in text-form not as a table a have to use it as it is.

OK, I pack all data into a <p> </p> with style white-space: pre-warp
and it works just fine

 

But in one case it does not

White spaces are all retained and with monospace font everything is fine,

But if the line starts with a whitespace all whitespaces in that line bevor the first character are discarded.

When I use the html code in a browser, the whitespaces are there.

I fixed it for me, by replacing it.
If the line starts with a white space I replace the first one with a <span>&nbsp;</span>
that works. Only replacing the space(s) with &nbsp; without the <span> does not work eigther.

So no problem for me, but just for your information.


I hope you understand what I mean. I am sorry, but English is not my native language.

best regards

Mirko

 

 

 

 

 

Von: Richard Hewitson ***@***.***> 
Gesendet: Donnerstag, 27. Januar 2022 13:35
An: richard-scryber/scryber.core ***@***.***>
Cc: TheRealNapalm ***@***.***>; Author ***@***.***>
Betreff: Re: [richard-scryber/scryber.core] RowSpan (Discussion #65)

 

Hi Mirko

Thanks, really glad you like Scryber.

I do hope we can support rowspan in the future - it just hurts my head at the moment, to get the calculations sorted. :-)

At the moment, nested tables may work for you, but the overflow onto new pages could be a blocker.
If there is anything else you can think would make Scryber better, be lovely to hear.

Kind regards

Richard

—
Reply to this email directly, view it on GitHub <https://github.com/richard-scryber/scryber.core/discussions/65#discussioncomment-2058363> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACS72I6IOK6XN7AUAPG33GDUYE3X3ANCNFSM5MXRKBEQ> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .

**Comment by richard-scryber** [comment]
Thanks Mirko and I have a feeling your English is far better than anything I could attempt in your native language.

However - I think this is a good catch and should be a pretty easy fix. Thanks for picking up #
